docs/core/linux-prerequisites.md

```diff
@@ -127,244 +155,417 @@ scl enable rh-dotnet20 bash
     sudo apt-get update
     ```
+
     **Ubuntu 16.04 / Linux Mint 18**
+
     ```bash
-    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
+
     **Ubuntu 16.10**
+
     ```bash
-    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list'
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
-2. Install .NET Core.
-# [.NET Core 2.x](#tab/netcore2x)
 
-After host package feed setup, install .NET Core 2.0 on Ubuntu or Linux Mint:
-```bash
-sudo apt-get install dotnet-sdk-2.0.0
-```
-# [.NET Core 1.x](#tab/netcore1x)
+3. Install .NET Core 1.x on Ubuntu or Linux Mint:
+
+    ```bash
+    sudo apt-get install dotnet-dev-1.0.4
+    ```
+
+4. Run the `dotnet --version` command to prove the installation succeeded.
+
+    ```bash
+        dotnet --version
+    ```
+
+---
+
+ ## Install .NET Core for Debian 8 or Debian 9 (64 bit)
+
+To install .NET Core on Debian 8 or Debian 9 (64 bit):
+
+1. Remove any **previous preview** versions of .NET Core from your system.
+
+2. Get the prerequisites.
 
-After host package feed setup, install .NET Core 1.x on Ubuntu or Linux Mint:
 ```bash
-sudo apt-get install dotnet-dev-1.0.4
+sudo apt-get install curl libunwind8 gettext
 ```
----
-3. Run the `dotnet --help` command to prove the installation succeeded.
 
- ```bash
-     dotnet --help
- ```
-## Install .NET Core for Debian 8 or 9 (64 bit).
+> [!Note]
+> A user-controlled directory is required for Linux system installs from tar.gz.
+
+# [.NET Core 2.x](#tab/netcore2x)
+
+3. Download the .NET Core SDK binaries (tarball).
+
+    ```bash
+    curl -sSL -o dotnet.tar.gz https://aka.ms/dotnet-sdk-2.0.0-linux-x64
+    ```
+1. Extract the .NET Core SDK binaries.
+
+    ```bash
+    mkdir -p ~/dotnet && tar zxf dotnet.tar.gz -C ~/dotnet
+    ```
 
-> [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+5. Add dotnet to your PATH.
 
-To install .NET Core on Debian 8 or 9 (64 bit):
-1. Get the prerequisites. 
     ```bash
-    sudo apt-get install curl libunwind8 gettext
+    export PATH=$PATH:$HOME/dotnet
     ```
-2. Download the .NET Core SDK binaries (tarball).
 
-# [.NET Core 2.x](#tab/netcore2x)   
+# [.NET Core 1.x](#tab/netcore1x)
+
+3. Download the .NET Core SDK binaries (tarball).
 
 ```bash
```

Comment:
use indentation here as you did for the 2.0 version. it looks better.

![image](https://user-images.githubusercontent.com/12971179/29892454-726f0e48-8d83-11e7-914a-30dd2eebbf82.png)

vs. 

![image](https://user-images.githubusercontent.com/12971179/29892474-7e7db522-8d83-11e7-9508-962dd7f637fc.png)